programs/server/Server.cpp

```diff
@@ -1650,9 +1650,9 @@ try
         database_catalog.initializeAndLoadTemporaryDatabase();
         loadMetadataSystem(global_context);
         maybeConvertSystemDatabase(global_context);
-        startupSystemTables();
         /// After attaching system databases we can initialize system log.
         global_context->initializeSystemLogs();
+        startupSystemTables();
         global_context->setSystemZooKeeperLogAfterInitializationIfNeeded();
```

Comment:
This change will re-introduce a race condition between the system database initialization and creation of new tables in the database